roles/container-engine/kata-containers/molecule/default/molecule.yml

```diff
@@ -0,0 +1,51 @@
+---
+driver:
+  name: vagrant
+  provider:
+    name: libvirt
+    options:
+      driver: kvm
+lint: |
+  set -e
+  yamllint -c ../../../.yamllint .
+platforms:
+  - name: ubuntu16
```

Comment:
Just a question: Ubuntu 16.04 will be end of life in 5 months.
Do we need to start testing it at this time?

Reply:
Changed to ubuntu1804